index.html

```diff
@@ -353,8 +356,38 @@ <h1 class="msg2">About <br> Myself</h1>
            </div>
           </section>
 
- </div>
 
+          <section class="form-section">
+            <div class="greet" font-weight="bold">
+                <h1 class="msg">Contact me</h1> 
+            </div>
+            <p class="info">If you have an application you are interested in developing,
+                          a feature that you need built or a project that needs coding.
+                          I’d love to help with it </p>
+        
+            <div class="form">
+                <form class="form-details">
+                   <ul>
+                        <li >
+                            <input name="name" class="name-input" type="name" placeholder="Yeremias" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input class="email-input" name="Email" type="email" placeholder="natajayanj14@gmail.com" required>
+                        </li>
+                        <li>
+                            <textarea class="msg-input" name="Email" type="textarea" placeholder="Write your message here" maxlength="500" required></textarea>
+                        </li>
+                        <li>
+                            <button class="button" type="submit">Get It Touch </button>
```

Comment:
- [ ] Great job working on the design but I think it would be better if you fix the image here to match the figma design as much as possible 🚀💯🔥📌

- Your design
![image](https://user-images.githubusercontent.com/58553711/199238969-2e6518e1-b165-4aab-9aa4-5dea54ff3cb0.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/199239170-2ec4d5fb-f7d6-49da-b2f0-10cb7ece2117.png)

Reply:
@Reem-lab I think the form is adjustable it can be manipulated locally to fit the desired size and shape